src/Make.sh

```diff
@@ -55,6 +56,10 @@ style () {
   fi
 }
 
+if (test $1 = "date") then
+  echo "static const char lammps_compile_date[] = \"`date`\";" > lmpcompiledate.h
```

Comment:
Is that what `__DATE__` and `__TIME__` is for?

Reply:
yes and no. `__DATE__` and `__TIME__` record the date and time when the given file was compiled. creating the include file forces recompilation of `lammps.cpp` and thus goes a step further and ensures this is the date/time when you last called `make`. since there are not frequent changes to `lammps.cpp` during normal development, the date/time output may be misleading. not sure if steve is ok with this, though. will have to discuss.